front/src/modules/trainschedule/components/Timetable/Timetable.tsx

```diff
@@ -66,6 +68,9 @@ export default function Timetable({
 
   const [deleteTrainSchedules] = osrdEditoastApi.endpoints.deleteTrainSchedule.useMutation();
 
+  const { updateTrainScheduleIDsToModify } =
+    useOsrdActions() as operationalStudiesConfSliceActionsType;
+
```

Comment:
Could you group it with the other custom hook line 54 :)

Reply:
Done